src/03-modality-agnostic-files.md

```diff
@@ -396,6 +396,284 @@ any identifiable information with the source code.
+
+iii. If provenance records are included, these records of provenance of a dataset 
+or a file MUST be described using a `[<prefix>_]prov.jsonld` file. 
+Since these [jsonld](https://json-ld.org/) documents are graph objects,
+ they can be aggregated without the need to apply any inheritance principle. 
+
+iv. The provenance file MAY be used to reflect the provenance of a dataset, 
+a collection of files or a specific file at any level_of the bids hierarchy. 
+
+v. Provenance information SHOULD be anonymized/de-identified as necessary. 
+
+### Examples of provenance in BIDS. 
+
+ 1. The raw conversion from DICOM images or other instrument native formats 
+to BIDS structure, details of stimulus presentation and cognitive paradigms, 
+and clinical and neuropsychiatric assessments, each come with their 
+own details of provenance.
+   ```
+    { "identifier": "sub-01/anat/..._T1.nii.gz",
+      "type": "NIfTIGZ",
+      "checksum": {"type": "sha512",
+                   "value": "21231221ab4534..."
+                  },
+      "derivedFrom": ["sourcedata/12345-1.dcm", "sourcedata/12345-2.dcm"],
+      "generatedBy": {"started": 2019-01-10T10:00:00"
+                      "associatedWith": {"type": "softwareAgent",
+                                         "name": "dcm2niix",
+                                         "version": "2.0.0"},
+                      "commandLine": "dcm2niix ..."
+                      }
+    }
+which inputs from the BIDS dataset were used together with what software was 
+run in what environment and with what parameters.
+   ```
+    { "identifier": "derivatives/freesurfer/sub-01/mri/orig/001.mgz",
+      "type": "MGZ",
+      "checksum": {"type": "sha512",
+                   "value": "121231221ab4534..."
+                  },
+      "derivedFrom": "sub-01/anat/..._T1.nii.gz",
+      "generatedBy": {"started": 2019-01-10T10:00:00"
+                      "associatedWith": {"type": "softwareAgent",
+                                         "name": "FreeSurfer",
+                                         "uri": "RRID:SCR_001847",
+                                         "version": "6.0.0"},
+                      "commandLine": "mri_convert ..."
+                      }
+    }
+involved in a study.
+   ```
+    {
+      "@context": "https://some/url/to/bids_context.jsonld",
+      "identifier": "http://example.org/ds00000X",
+      "generatedBy": {
+        "type": "Project",
+        "uri": "https://banda.mit.edu/",
+        "startedAt": "2016-09-01T10:00:00",
+        "wasAssociatedWith": { 
+          "type": "Organization",
+          "uri": "NIH",
+          "role": "Funding"},
+        },
+      "wasAttributedTo": { 
+         "type": "Person",
+         "name": "Prof. Smith",
+         "uri": "ORCID:0123",
+         "role": "PI"}
+       }
+    }
+appropriate attribution to the original dataset generators as well as 
+future transformers.  
+5. For datasets and derivatives, provenance can also include details of 
```

Comment:
There's something unclosed here so all the following text is reading like in the example/comment grey mode. Maybe it's just missing ``` to close the type of text or something similar. I'm not super expert in this formatting :)

Reply:
Also the numbering goes from 1 to 5, so maybe something missing in the middle?